data/scenarios/README.md

```diff
@@ -0,0 +1,214 @@
+
+### Entities
+
+The top-level `entities` field contains a list of entity descriptions.
+Each entity description is a key-value map described by the following
+table.
+
+| Key            | Default? | Type          | Description                                                                                                                                                                                                                                                                      |
+|----------------|----------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `name`         |          | `string`      | The name of the entity.  This is what will show up in the inventory and how the entity can be referred to.                                                                                                                                                                       |
+| `display`      |          | `map`         | [Display](#display) information for the entity.                                                                                                                                                                                                                                  |
+| `plural`       | `null`   | `string`      | An explicit plural form of the name of the entity.  If omitted, standard heuristics will be used for forming the English plural of its name.                                                                                                                                     |
+| `description`  |          | `string list` | A description of the entity, as a list of paragraphs.                                                                                                                                                                                                                            |
+| `orientation`  | `null`   | `int × int`   | A 2-tuple of integers specifying an orientation vector for the entity. Currently unused.                                                                                                                                                                                         |
+| `growth`       | `null`   | `int × int`   | For growable entities, a 2-tuple of integers specifying the minimum and maximum amount of time taken for one growth stage.  The actual time for one growth stage will be chosen uniformly at random from this range; it takes two growth stages for an entity to be fully grown. |
```

Comment:
AFAIK, we use a heterogenous list for pair and check that it in this case contains two entries that are both integers.

I think that needs an explanation somewhere. :confused:

Reply:
Good point!  I added a section explaining some conventions.